Improvement of docker top Man Page

pospispa opened the pull request:
Some users expect that the `docker top $CONT` command displays information from the inside container perspective.
They expect that the `docker top $CONT` command displays same information as the `docker exec $CONT ps -ef` command. But it does not.

That's why the `docker top` man page shall explicitly state that the `docker top $CONT` displays information from the host's point of view.

GordonTheTurtle commented:
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "Improvement-of-docker-top-Man-Page" git@github.com:pospispa/docker.git somewhere
$ cd somewhere
$ git commit --amend -s --no-edit
$ git push -f
~~~

Ammending updates the existing PR. You **DO NOT** need to open a new one.

vdemeester commented:
ping @thaJeztah @moxiegirl

moxiegirl commented on man/docker-top.1.md:
```diff
@@ -11,8 +11,9 @@ CONTAINER [ps OPTIONS]
- options you would pass to a Linux ps command.
+Display the running process of the container. ps-OPTION can be any of the options you would pass to a Linux ps command.
+
+All displayed information are from host's point of view.
```
@vdemeester  information `is` --- treated as singular in English --- unlike French and German.

pospispa replied:
Thank you for the correction. I updated the pull-request accordingly.

On Tue, Dec 1, 2015 at 6:03 PM, moxiegirl <notifications@github.com> wrote:

> In man/docker-top.1.md
> <https://github.com/docker/docker/pull/18240#discussion_r46305423>:
>
> > @@ -11,8 +11,9 @@ CONTAINER [ps OPTIONS]
> >
> >  # DESCRIPTION
> >
> > -Display the running process of the container. ps-OPTION can be any of the
> > - options you would pass to a Linux ps command.
> > +Display the running process of the container. ps-OPTION can be any of the options you would pass to a Linux ps command.
> > +
> > +All displayed information are from host's point of view.
>
> @vdemeester <https://github.com/vdemeester> information is --- treated as
> singular in English --- unlike French and German.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/18240/files#r46305423>.
>



-- 
Velkoformatovy pigmentovy inkoustovy tisk
Cerveny kopec 2a
639 00 Brno
tel.: +420 721 209 607
e-mail: pospispa@gmail.com
skype: pospispa

Royalty-free photographs:
http://refer.istockphoto.com/ta.php?lc=059498042431004651&atid=43021%7CBannerID%3D43021%7CReferralMethod%3DLink&url=http%3A%2F%2Fwww.istockphoto.com%2Fsearch%2Fportfolio%2F330814%2Fsource%2Fref

moxiegirl commented:
LGTM @pospispa this needs a rebase before it can be merged.

thaJeztah commented:
@moxiegirl we're investigating but looks like enabling "protected branches" on GitHub also had a nasty side-effect where every PR is marked "is-out-of-date" :cry: We've already sent an e-mail to GitHub because there were some other things as well :/

Rebasing does work though

pospispa commented:
Rebased.

On Wed, Dec 2, 2015 at 5:55 PM, Sebastiaan van Stijn <
notifications@github.com> wrote:

> @moxiegirl <https://github.com/moxiegirl> we're investigating but looks
> like enabling "protected branches" on GitHub also had a nasty side-effect
> where every PR is marked "is-out-of-date" [image: :cry:] We've already
> sent an e-mail to GitHub because there were some other things as well :/
>
> Rebasing does work though
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/18240#issuecomment-161362667>.
>



-- 
Velkoformatovy pigmentovy inkoustovy tisk
Cerveny kopec 2a
639 00 Brno
tel.: +420 721 209 607
e-mail: pospispa@gmail.com
skype: pospispa

Royalty-free photographs:
http://refer.istockphoto.com/ta.php?lc=059498042431004651&atid=43021%7CBannerID%3D43021%7CReferralMethod%3DLink&url=http%3A%2F%2Fwww.istockphoto.com%2Fsearch%2Fportfolio%2F330814%2Fsource%2Fref

pospispa committed:
Improvement of docker top Man Page

Some users expect that the `docker top $CONT` command displays information from the inside container perspective.
They expect that the `docker top $CONT` command displays same information as the `docker exec $CONT ps -ef` command. But it does not.

That's why the `docker top` man page shall explicitly state that the `docker top $CONT` displays information from the host's point of view.

Signed-off-by: Pavel Pospisil <pospispa@gmail.com>

thaJeztah commented:
LGTM, thanks @pospispa

calavera commented:
LGTM
